website/src/components/quickstartGuideList/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Head from '@docusaurus/Head';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import Layout from '@theme/Layout';
+import Hero from '@site/src/components/hero';
+import QuickstartGuideCard from '../quickstartGuideCard'
+import styles from './styles.module.css';
+
+const quickstartTitle = 'Quickstarts'
+const quickstartDescription = 'dbt Core is a powerful open-source tool for data transformations and dbt Cloud is the fastest and most reliable way to deploy your dbt jobs. With the help of a sample project, learn how to quickly start using dbt and one of the most common data platforms.'
```

Comment:
adding dbt cloud first before dbt core and also adding longer description, but feel free to cut it if it's too long 

```suggestion
const quickstartDescription = 'dbt Cloud is the fastest and most reliable way to deploy your dbt jobs and dbt Core is a powerful open-source tool for data transformations. 

With the help of a sample project, these quickstarts will quickly show you how to set up dbt with some of the most common data platforms and perform some key tasks. These tasks will illustrate how dbt establishes standard practices for your work such as using Git to version control your analytics code, testing your models to validate they are working correctly, documenting your models in detail to better collaborate with others, and scheduling jobs to execute at designated times.'
```

Reply:
also @john-rock hey @john-rock , i think is almost good to go on my end! one more thing I'm curious about:

is it possible to sort the cards so that cloud quickstart are together and then core quickstart are together? this is to make it easier for the user to select their path:

- the first row can be Cloud with BigQuery, Cloud with Databricks, Cloud with Redshfit.
- 2nd row is Cloud with Snowflake, Cloud with Starburst, Core with Codespace
- 3rd row is Core with manual install.

Reply:
Hey @mirnawong1! Thanks for the feedback. 

Most of the sorting enhancements will be included in a V2 of this. Unfortenly right now I won't have the capacity to apply directly what you are suggesting above.

What I can do, and this is now available in the preview, is add a frontmatter key (`platform`) and added a conditional check to sort all quickstarts with the frontmatter `platform = 'dbt-cloud'` first following by the `dbt-core` quickstarts.

Let me know if you are okay with that solution for now.

Reply:
As for the content in the hero section, right now that component is setup to only accept a string for the value of the `subheading`. Because of this we currently can not add any html as it would always be rendered as a string.

Here's a screenshot of your suggested text below. IMO, it does look a bit long but let me know what you think.

![image](https://github.com/dbt-labs/docs.getdbt.com/assets/46692803/f210d12a-5318-4a54-b68c-17ea49c8d069)

Reply:
hey @john-rock, i’ll remove/update my content suggestion as youre so right, it looks too long!